Updated MacOS usb-serial info for the T-Echo

ventz opened the pull request:
* Specifically around the usb serial/TTY information.
* A note about not needing (and actually having to remove the drivers/kernel extension)

sachaw commented on website/docs/getting-started/flashing-nrf52.md:
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 
+#### Mac OS
+
+(Verified as of 11/2021 on the latest T-Echo nRF52840 on Mac OS Monterey v12.0.1 - Intel chipset)
```
Don't need a timestamp, as these steps should be as timeless as possible.

ventz replied:
Done - cleaned up (overwrote latest push - see new file)

sachaw commented on website/docs/getting-started/flashing-nrf52.md:
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
 ## Pre-requisits
 
-Please ensure that you use a data USB cable, as some cables provide power only and not the data lines.
+Please ensure that you use a data USB cable, as some cables provide power only and not the data lines. Many popular brands (ie: Anker) are also "power" only.
```
Not really needed, inferred by previous comment.

ventz replied:
Done - cleaned up (overwrote latest push - see new file)

sachaw commented on website/docs/getting-started/flashing-nrf52.md:
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
+
+If you have it, you can use the Apple (iPad Pro included) usb-C to usb-C cable, as it has "data" lines. The cable provided from LILYGO will not work, as it's power only.
+
+CRITICAL: *DO NOT* download the USB device drivers - this will actually prevent you from connecting to your T-Echo from your Mac.
```
This is a good place for an admonition:
https://docusaurus.io/docs/markdown-features/admonitions

ventz replied:
Done - added + cleaned up (overwrote latest push - see new file)

sachaw commented on website/docs/getting-started/flashing-nrf52.md:
```diff
@@ -8,12 +8,44 @@ import TabItem from '@theme/TabItem';
+
+If you have already downloaded/installed the MacOS WCH-IC CH340 ("CH341SER_MAC") drivers via the `CH34x_Install_V1.5.pkg`, you will have to *uninstall* the kernel extension:
+
+* Unplug your T-Echo
+* Open the Terminal and run: `sudo -rf /Library/Extensions/usbserial.kext`
+* Reboot
+
+If you are starting clean (or have removed the WCH USB driver), verify successful connections with 
+
+* Plug in your T-Echo
+* Open the Terminal
+* `ls -l /dev/tty.usbmodem*`
+
+If the device file exists, you will also notice a "TECHOBOOT" volume in the Finder, or in the Terminal after double-clicking the Reset Button (see below) 
+
+* `ls /Volumes/TECHOBOOT`
```
Might be a good case to use `details`: https://docusaurus.io/docs/markdown-features#details
for edge cases as users are most likely to want to read them if they are looking for it in the first place.

ventz replied:
Done - added detail block (overwrote latest push - see new file)

sachaw commented on website/docs/getting-started/flashing-nrf52.md:
```diff
@@ -33,6 +65,9 @@ Be careful to install the correct load for your board. While it is unlikely that
+
 * A new drive will then be mounted on your computer. Open this drive and you should see three files: `CURRENT.UF2`, `INDEX.HTM`, and `INFO_UF2.TXT`
-* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive.
+* Copy the appropriate `firmware-xxxxx-1.2.x.uf2` file from the firmware zip file onto the new drive. NOTE: You are going to copy/drop "as is" 'firmware-xxxxx-1.2.x.uf2' (NOT over "CURRENT.UF2") in the volume, and the device reboot will copy it/load it correctly.
```
Another place for an admonition.

ventz replied:
Done - added (overwrote latest push - see new file)

sachaw reviewed (changes requested):
Looks good, just a few pointers to fix up, then I can merge.

ventz commented:
@sachaw Thanks for the detailed feedback. Cleaned up, updated (with admonitions and details), and re-pushed (force updated since my push is not used upstream anywhere yet)
